### PR TITLE
docs(cardinal): Add persona tag section to docs

### PR DESCRIPTION
**Pull request by jerargus**
Closes: WORLD-625

## Overview

Add a section to the Cardinal docs covering Persona Tag. 

There's also a section in the Nakama docs about Persona Tags, however that section focuses on how Nakama sends Persona Tag requests to Cardinal.

## Testing and Verifying

Doc changes only; no tests.

### CODE REVIEW

File: docs/pages/Cardinal/persona-tags.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Your Cardinal instance will only process Transactions that have been signed by a Persona Tag that has been previously registered with your Cardinal instance. Registering a Persona Tag ensures other people or smart contracts cannot act on your behalf when interacting with the Cardinal backend.
+
+## ECS Integration
+
+Persona Tag management is automatically included with Cardinal games. A system (called AuthorizePersonaAddressSystem) is registered with your world on startup, and a MessageType (called AuthorizePersonaAddressMsg) is used to track new Persona Tag creation requests. This registration happens automatically so nothing extra needs to be done to start processing Persona Tag creation requests.
```
Comment (technicallyty) [inline docs/pages/Cardinal/persona-tags.mdx:9]: should be CreatePersona here

File: docs/pages/Cardinal/persona-tags.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		personaTag := attack.Tx().PersonaTag
+		worldCtx.Logger().Debug().Msgf("The persona tag is %q", personaTag)
+		return AttackOutput{}, nil
+	})
+}
+```
+
+## Nakama
+
+The easiest way to set up a Persona Tag with your cardinal game it to use Nakama. The `/nakama/claim-persona` RPC endpoint takes a request with a body of:
```
Comment (technicallyty) [inline docs/pages/Cardinal/persona-tags.mdx:46]: i think we should specify further abt nakama - as in its not just Nakama itself that provides this, but our custom Nakama impl

File: docs/pages/Cardinal/persona-tags.mdx (hunk 1 above)
Comment (jerargus) [reply docs/pages/Cardinal/persona-tags.mdx]: Fixed

File: docs/pages/Cardinal/persona-tags.mdx (hunk 2 above)
Comment (jerargus) [reply docs/pages/Cardinal/persona-tags.mdx]: I call out the Cardinal/Nakama plugin, and I add a link to the docs with more details. Let me know if you think I should expand more on this section in addition to linking to the plugin page.

Comment (technicallyty) [reply docs/pages/Cardinal/persona-tags.mdx]: looks good!

**Review by technicallyty — APPROVED** [review]

**Pull request merged**